### PR TITLE
Support for posting measurement tags and connecting to InfluxDB using SSL

### PR DESCRIPTION
**Pull request by ayoy**
New parameters:
* `tags` - hash with measurement tags that can be posted alongside values in `write_point`
* `use_ssl`, `verify_ssl` - passed to `InfluxDB::Client.new`, by default `use_ssl = false` and `verify_ssl = true` (but it's ignored when `use_ssl` is set to `false`).

### CODE REVIEW

**Comment by giginet** [comment]
@ayoy I'm so sorry. I missed this until now 🙏

File: lib/fastlane/plugin/influxdb/actions/influxdb_action.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 require 'influxdb'
 
+module Bool; end
```
Comment (giginet) [inline lib/fastlane/plugin/influxdb/actions/influxdb_action.rb]: It is not necessary. fastlane-tools already provide `Boolean` type

File: lib/fastlane/plugin/influxdb/actions/influxdb_action.rb [hunk 2]
```diff
@@ -55,6 +63,18 @@ def self.available_options
                                   optional: false,
                              default_value: 8086,
                                       type: Integer),
+          FastlaneCore::ConfigItem.new(key: :use_ssl,
+                                  env_name: "INFLUXDB_USE_SSL",
+                               description: "Whether or not to use SSL (HTTPS)",
+                                  optional: true,
+                             default_value: false,
+                                      type: Bool),
```
Comment (giginet) [inline lib/fastlane/plugin/influxdb/actions/influxdb_action.rb]: You can use `Boolean`

File: lib/fastlane/plugin/influxdb/version.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Influxdb
-    VERSION = "0.1.0"
+    VERSION = "0.2.1"
```
Comment (giginet) [inline lib/fastlane/plugin/influxdb/version.rb]: You can bump up version to 0.3.0 because of adding new parametes.

**Comment by ayoy** [comment]
@giginet understood, it happens to me too :) I've been successfully using my fork in the meantime. 
Let me just fix the dirty Boolean hack I introduced there.

**Comment by ayoy** [comment]
@giginet sorry for the mess, I have followed the docs on fastlane with regard to boolean parameters:

https://docs.fastlane.tools/advanced/#boolean-parameters

and set `is_string` to `false`, instead of using the Boolean type. I believe the version should remain at 0.2.0 since it's the iteration on top of a 0.1.0. If you prefer to have it done your way, let me know and I'll do the changes.

**Comment by giginet** [comment]
@ayoy I updated dependencies and fixed some CI errors on #2.
Could you rebase current master and you exec `bundle exec rubocop -a`?

**Comment by ayoy** [comment]
@giginet done, all good!

**Comment by giginet** [comment]
I just released v0.2.0 on RubyGems.
Sorry for waiting you!

**Pull request merged**